share/spack/docker/ubuntu-2010.dockerfile

```
@@ -0,0 +1,76 @@
+FROM ubuntu:20.10
```

Having a recent Ubuntu is great, thanks! Wondering if we should:
```suggestion
FROM ubuntu:20.04
```
to have all our Spack + Ubuntu images based on LTS.